src/test/java/br/com/quality/desafio_quality/service/DistrictServiceTest.java

```diff
@@ -0,0 +1,112 @@
+package br.com.quality.desafio_quality.service;
+
+import br.com.quality.desafio_quality.entity.District;
+import br.com.quality.desafio_quality.repository.DistrictRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+public class DistrictServiceTest {
+    private DistrictService service;
+    private DistrictRepository repository;
+
+    @BeforeEach
+    void init() {
+        repository = Mockito.mock(DistrictRepository.class);
+        service = new DistrictService(repository);
+    }
+
```

Comment:
Cria um método para testar o construtor sem argumentos, algo do tipo:
```suggestion
@Test
  public void shoulReturnAnInstanceOfDistrictService() {
    DistrictService service = new DistrictService();

    Assertions.assertNotNull(service);
  }
```